src/actions/actionUnbindText.tsx

```diff
@@ -0,0 +1,45 @@
+import { getNonDeletedElements } from "../element";
```

Comment:
why we want to move this to a new file ?

Reply:
bad merge. fixed